person.rb

```diff
@@ -1,3 +1,5 @@
+require './nameable'
+
 class Person
   attr_reader :id
```

Comment:
- [ ] The `Person` class should inherit from `Nameable` class. It also should have a method `correct_name` that returns the name attribute.